src/escpos/escpos.py

```diff
@@ -128,6 +132,27 @@ def __del__(self):
         """Call self.close upon deletion."""
         self.close()
 
+    @property
+    def device(self) -> Union[Literal[False], Literal[None], object]:
```

Comment:
```suggestion
    def device(self) -> Union[Literal[None], object]:
```
I think the return value cannot become False.

Reply:
mypy cli doesn't complain about `def device(self) -> Union[Literal[False], Literal[None], object]:`

I think `False` is not accepted but it is `Literal[False]`

Reply:
I just meant with my comment, that we could narrow down the types that are reported here. If I have a look at the code, I cannot see any way that there would ever be a literal False returned. Therefore we could "add" that as information.

Reply:
I misunderstood your comment, sorry.
You are right, device only returns None or the connection object.
Thank you very much!

Reply:
Thank you! 👍